File: PharmaDicBackEnd/PharmaDicBackEnd.ApiService/Models/MedicineWarning.cs
using System;
using System.Collections.Generic;

namespace PharmaDicBackEnd.ApiService.Models;

public partial class MedicineWarning
{
    public int WarningId { get; set; }

    public int MedicineId { get; set; }

    public string? WarningContent { get; set; }

    public string? WarningLevel { get; set; }

    public virtual Medicine Medicine { get; set; } = null!;
}

File: PharmaDicMobile/app/src/main/java/com/example/pharmadicmobile/models/MedicineWarning.java
package com.example.pharmadicmobile.models;

import java.io.Serializable;

public class MedicineWarning implements Serializable {
    private int warningId;
    private int medicineId;
    private String warningContent;
    private String warningLevel;
    private Medicine medicine;

    public MedicineWarning() {}

    public int getWarningId() { return warningId; }
    public void setWarningId(int warningId) { this.warningId = warningId; }

    public int getMedicineId() { return medicineId; }
    public void setMedicineId(int medicineId) { this.medicineId = medicineId; }

    public String getWarningContent() { return warningContent; }
    public void setWarningContent(String warningContent) { this.warningContent = warningContent; }

    public String getWarningLevel() { return warningLevel; }
    public void setWarningLevel(String warningLevel) { this.warningLevel = warningLevel; }

    public Medicine getMedicine() { return medicine; }
    public void setMedicine(Medicine medicine) { this.medicine = medicine; }
}
